content/en/docs/instrumentation/php/automatic.md

```diff
@@ -45,26 +45,39 @@ function ends it.
 
 ## Setup
 
-1. Install the extension via [pickle](https://github.com/FriendsOfPHP/pickle) or [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer) (docker specific):
+1.  Install the extension via [pickle](https://github.com/FriendsOfPHP/pickle)
+    or
+    [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer)
+    (docker specific):
 
-    * **pickle** can be used to install extensions that are available via http://pecl.php.net, however that's not the case
-    for opentelemetry-php-instrumentation yet, so the only way for it is to install directly from source code.
-    Following command line shows how to do that using specific version of extension (1.0.0beta1 in this case).
+    - **pickle** can be used to install extensions that are available via
+      http://pecl.php.net, however that's not the case for
+      opentelemetry-php-instrumentation yet, so the only way for it is to
+      install directly from source code. The following command line shows you
+      how to do that using a specific version of the extension (1.0.0beta2 in
+      this case):
 
-        ```console
-        $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta1
-        ```
-   
-   *  **php-extension-installer**
-        ```console
-        $ install-php-extensions open-telemetry/opentelemetry-php-instrumentation@main
-        ```
+          ```console
```

Comment:
Seems that prettier messed up formatting

Reply:
Well yes and no. The formatting was a bit off / odd already.

Reply:
FYI, I committed a fix, which was just to unindent the two code blocks.

Reply:
thx.